x-pack/plugins/secops/server/lib/hosts/types.ts

```diff
@@ -34,18 +34,16 @@ export interface HostHit {
     '@timestamp': string;
     system: {
       host: {
+        name: string;
         os: {
           name: string;
           version: string;
         };
       };
     };
-    host: {
-      name: string;
-    };
   };
   cursor: string;
-  sort: [string | number];
+  sort: string[] | number[];
```

Comment:
I do not think that the same thing. that was there because the array had string or number

Reply:
It's a subtle diff between a typescript tuple and a typescript array:

[string] = Tuple (fixed size array)
string[] = Array (most common array)

https://mattferderer.com/difference-between-array-types-in-typescript

Reply:
if I remember correctly, I think that we want more something like that:
```
type StringOrNumber = string | number;
=> sort: StringOrNumber[]```